core/src/com/unciv/logic/city/CityStats.kt

```diff
@@ -268,6 +270,14 @@ class CityStats {
                 val amountOfEffects = (cityInfo.population.population / unique.params[1].toInt()).toFloat()
                 stats.add(unique.stats.times(amountOfEffects))
             }
+            
+            // "[stats] in cities with [amount] or more population
+            if (unique.placeholderText == "[] in cities with [] or more population" && cityInfo.population.population >= unique.params[1].toInt())
+                stats.add(unique.stats)
+            
+            // "[stats] from cities on [tileFilter] tiles"
+            if (unique.placeholderText == "[] from cities on [] tiles" && cityInfo.getCenterTile().matchesTerrainFilter(unique.params[1]))
```

Comment:
"In cities" sound better that "from cities" to me, you added 2 unique with different phrasings